React Task/src/components/Buynow.jsx

```diff
@@ -0,0 +1,214 @@
+import React from 'react'
+import { useParams } from 'react-router'
+import DATA from '../Data';
+import { useEffect } from 'react';
+
+const Buynow = () => {
+    useEffect(() => {
+        window.scrollTo(0, 0)
+    }, [])
+    const proid = useParams();
+    const proDetail = DATA.filter(x => x.id == proid.id)
```

Comment:
Move data filter under use effect 

Empty dependency array means that the hook will only trigger once when the component is first rendered.